application/controllers/logic.py

```diff
@@ -52,81 +68,93 @@ def check_move (card, player, is_discarded, for_wonder):
         return False
     else:
         return True
-"""
 
-def process_card(card, player, is_discarded, for_wonder):
-    # checks move is valid, returns false if not
-    #if not check_move(card, player, is_discarded, for_wonder):
-    #    return False
-    if is_discarded:
-        player.money += 3
 
-        history = Cardhist(playerId=player.id, cardId=card.id, discarded=True)
-        db.session.add(history)
+def update_player(card, player):
```

Comment:
Is player and card always an object here? I don't think this function is updating all of the player attributes at once are they?